contracts/Aavegotchi/libraries/LibGotchiLending.sol

```diff
@@ -116,6 +138,7 @@ library LibGotchiLending {
         address thirdParty;
         uint32 whitelistId;
         address[] revenueTokens;
+        uint256 channellingStatus;
```

Comment:
@Timidan What are the possible values of this, and what do they mean? You can add some documentation for that

Reply:
Also, I would recommend changing this to a more dynamic name, maybe "permissions".

0 = nothing
1 = channeling
2 = channeling + ?
3= channeling + 

Another approach would be instead of using uint256, we use an array. Currently we only have one permission though, but it would be best if thats a dynamic array

Reply:
i wouldn't recommend an array as that would add some overhead gas costs for `permission` changes

any changes would require a complex `pop` or push

whereas a single `uint256` value would just depict predefined behaviours

Reply:
also, it would be cumbersome to check for current permissions, especially if there are more permissions in the future, it'd need to copy the whole array content to memory.

making a change from `channelingStatus` to `permissions` will change the current abi...will sync with frank